docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -556,6 +556,12 @@ Size of the index in <<byte-units, byte units>>.
 
 `translog`::
 <<index-modules-translog,Translog>> statistics.
+
+`sparse_vector`::
```

Comment:
nitpick - but it looks like the rest of the metrics are in alphabetical order - so we should aim to maintain it?

Reply:
Good catch, fixed